src/GlobalStyle.ts

```diff
@@ -2,7 +2,7 @@ import { createGlobalStyle } from 'styled-components';
 import { normalize } from 'polished';
 import theme from './theme';
 
-const GlobalStyle = createGlobalStyle`
+const GlobalStyle: any = createGlobalStyle` // TODO: Wait Fix from @types/styled-component : https://github.com/styled-components/styled-components/issues/3738
```

Comment:
Has this been fixed? I tried removing the `: any` locally and it seems to build fine.

Looking at the `package.json`, it seems that we've applied this fix already: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/59765#issuecomment-1104510951

Reply:
hm... I have it in resolutions, will check again, nice catch!

Reply:
done